.github/workflows/pr-deploy.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
       - name: Get PR number, title, and branch name
         id: pr_info
         run: |
-          set -euxo pipefail
+          set -e pipefail
```

Comment:
Do you mean `set -eo pipefail`? `set -euxo pipefail` is short for `set -e -u -x -o pipefail` (`pipefail` is an argument to the `-o` flag)

Reply:
I wanted to remove -x only to not print everything
Is it fine to just keep `set -e`?

Reply:
Use `set -euo pipefail` in that case. `-x` is the only thing causing it to print. The rest make the script fail immediately on various errors that wouldn't cause the script to fail otherwise, which is what you want.